Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ts

```diff
@@ -37,6 +39,20 @@ async function run() {
         if (!tl.exist(webDeployPkg)) {
             throw new Error(tl.loc('Invalidwebapppackageorfolderpathprovided', webDeployPkg));
         }
+
+        var isParamFilePresentInPackage = false;
+        var isFolderBasedDeployment = !msDeployUtility.fileExists(webDeployPkg);
```

Comment:
We seem to be using MSDeployUtility. If we use msdeploy, then how can the Kudu flow be xPlat?